docker-compose.override.yml

```diff
@@ -0,0 +1,35 @@
+version: '3.8'
+
+services:
+
+  nginx:
+    image: nginx:alpine
+    depends_on:
+      - phpfpm
+    volumes:
+      - .:/application:cached
+      - ./sys/nginx/dev/default.conf:/etc/nginx/conf.d/default.conf:ro
+      - ./var/log/nginx/:/var/log/nginx:cached
+    ports:
+      - '8001:80'
+
+
+  phpfpm:
+    build:
+      context: .
+      dockerfile: sys/docker/alpine-phpfpm/dev/Dockerfile
```

Comment:
in this way, we might end up missing critical configurations, and be in one of those classic "it works on my pc" ;)